modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java

```diff
@@ -103,7 +119,8 @@ public PhpSymfonyServerCodegen() {
         // default HIDE_GENERATION_TIMESTAMP to true
         hideGenerationTimestamp = Boolean.TRUE;
 
-        setReservedWordsLowerCase(
+        // TODO Are those case sensitive or insensitive
+        super.registerReservedWordsCaseInsensitive(
```

Comment:
case insensitive

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)